inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseProfileDeserializationTest.java

```diff
@@ -40,6 +42,8 @@
     public void checkThatDeserializationWorks() throws JsonParseException, JsonMappingException, IOException {
         String name = "Test KB";
         String url = "http://someurl/sparql";
+        KnowledgeBaseAccessType accessType = KnowledgeBaseAccessType.CLASSPATH;
```

Comment:
Shouldn't that be "LOCAL" instead of "CLASSPATH"?

Reply:
The URL says it all. If the URL starts with `classpath:`, then we should fetch the inputstream for the data from via the Spring resolver. Otherwise, we should try to use Java URL and get the stream from there. Once the stream has been obtained, the code is basically the same.